docs/package.json

```diff
@@ -17,7 +17,7 @@
 	"dependencies": {
 		"@docusaurus/core": "^3.0.1",
 		"@docusaurus/preset-classic": "^3.0.1",
-		"@docusaurus/theme-live-codeblock": "^3.0.1",
+		"@docusaurus/theme-live-codeblock": "3.0.1",
```

Comment:
why `@docusaurus/theme-live-codeblock` version is fixed?

Reply:
I've updated it back to "^3.0.1"